### ISSUE TITLE
Add a possibility to filter by lexical category

### ISSUE THREAD

**Issue by belett** [Vesihiisi/Wikidata-Senses#19]
Hi,

It would be nice to be able to filter by lexical category.

In particular, and at least for me, adding senses make more sens (pun intended) and is easier for nouns, so I would like to see only nouns (especially as there should be few to none in Breton).

**Comment by Vesihiisi** [comment]
I agree this would be useful. Putting this on top of my horrible guilt queue.
